README.md

```diff
@@ -1,16 +1,56 @@
-# Title
+# What Makes an Effective Programmer?
 
-## Learning Goals
+What do the best programmers do that the average programmers don't?
 
--SWBAT 1
--SWBAT 2
+_Do they write more code faster?_
 
-## Introduction
+Can we say that because person A wrote 10,000 lines and person B wrote 500
+that A is outperforming B? No, we can't. In the world of software development
+500 lines of code can do the work of 10,000 and it's much easier to work on the
+500 lines than 10,000.
 
-## SWBAT 1
+_Do they get code to pass tests more quickly?_
 
-## SWBAT 2
+Sometimes! But since programmers write the tests as well as their implementation
+code—the code that satisfies the tests—they can only test what they themselves
+know about. An inexperienced programmer will write low-value tests that low-
+value code can pass quickly, and sleep soundly not knowing how blind they were.
+The measure of code is not passing already written tests, but creating the
+potential to expand on the code's behavior in the future.
 
-## Conclusion
+The difference is _process_.
 
-## Resources
+Effective programmers take the time to _really_ understand the problem
+they're trying to solve. Once they understand it, they make focused, decided
+steps with _mindfulness_ to implement the solution. **Learning to program is the
+art of solving problems.** It's the main muscle that you are constantly
+developing.
+
+Let's dive in a bit deeper here. What is problem solving?
+
+## Problem Solving in the Abstract
+
+Think about some of the problems people have on a regular basis. If you're a
+soccer player, getting past the defense to score a goal is a problem, because
+there are people blocking you from kicking the ball into the net. If you're
+hungry, making a peanut butter and jelly sandwich is a problem, because you need
+to find the ingredients and assemble them in the correct order. Whether we call
+it a "play" on the soccer field or a "recipe" in the kitchen, essentially what
+you're doing is following a process to create _another_ process &mdash; a
+process that will solve the problem.
+
+Good programmers do this every time they start on a new bit of work. From adding
+a little updated feature to creating a brand-new project from scratch. Their
+process typically looks like:
+
+* What is the problem I'm trying to solve?
+* What does the output look like? What is the end result I'm trying to build?
+* What inputs do I need in order to create that output?
+* What do I have now? What is missing?
+
+### How Do We Develop the Problem-Solving Muscle?
+
+You build the problem-solving skill the same you build every skill: constant,
+_conscious_ effort and practice. The great thing about learning to program is
+that you have constant access to problems that help build this skill. Every lab
+and every real-world observation is an opportunity to practice and get better.
```

Comment:
This is a great point. I feel like it could be emphasized even more.

I think also adding a conclusion to resummarize will help tie the concepts together